packages/material-icon/README.md

```diff
@@ -8,18 +8,17 @@ MDC React Material Icon is a component for material icons.
 npm install @material/react-material-icon
 ```
 
-## Usage
-
-### JSX Structure
-
-```html
-<MaterialIcon />
-```
-
-
 ## Props
 
 Prop Name | Type | Description
 --- | --- | ---
 icon | string | Type of icon to be displayed.
 className | string | Classes to pass on to the root `<i>` element.
+hasRipple | n/a | If present on element, it will enable the ripple on the icon.
+
+
+## CSS Classes
+
+CSS Class | Description
+--- | ---
+`material-icons--ripple-surface` | Adds ripple to the icon.
```

Comment:
This is an internal implementation detail. you dont need to add it to the README